criu/cr-restore.c

```diff
@@ -1846,6 +1847,11 @@ static int restore_task_with_children(void *_arg)
 				goto err;
 		}
 
+		if (parse_pid_status(PROC_SELF, &creds.s, NULL))
+			goto err;
+
+		memcpy(&opts.cap_eff, &creds.cap_eff, sizeof(u32) * PROC_CAP_SIZE);
```

Comment:
what is going on here? why can we not use capget? Pls write a comment here.

Reply:
I really don't know. I just used what CRIU already provided. If capget() is better, I can use that.

Reply:
Seeing the old comments I see that @nviennot suggested reading /proc/self/status and as there is a function for that (`parse_pid_status()`) I was using that. I have no strong opinion which was one to use. `capget()` is nicer as it does not require string parsing, but just let me know what is recommended.

Reply:
capget() is much faster and more reliable.

Reply:
switched to capget